gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -1047,6 +1336,8 @@ gst_tensor_src_iio_stop (GstBaseSrc * src)
 
   self->configured = FALSE;
 
+  g_free (self->tensors_config);
```

Comment:
This line also needs free function for config-name)

Reply:
Done.